.github/workflows/check-easyjson.yml

```
@@ -0,0 +1,58 @@
+name: Check easyjson generated files
+
+env:
+  # See: https://github.com/actions/setup-go/tree/v2#readme
+  GO_VERSION: "1.17"
+
+# See: https://docs.github.com/en/actions/reference/events-that-trigger-workflows
+on:
+  push:
+    paths:
+      - ".github/workflows/check-easyjson.ya?ml"
+      - "codecov.ya?ml"
+      - "go.mod"
+      - "go.sum"
+      - "Taskfile.ya?ml"
+      - "**.go"
+  pull_request:
+    paths:
+      - ".github/workflows/check-easyjson.ya?ml"
+      - "codecov.ya?ml"
```

```suggestion
      - "go.mod"
      - "go.sum"
      - "Taskfile.ya?ml"
      - "**.go"
  pull_request:
    paths:
      - ".github/workflows/check-easyjson.ya?ml"
```
As far as I can tell, it should not be necessary to trigger the workflow on a change to the Codecov configuration file.